src/App.js

```diff
@@ -1,23 +1,24 @@
-import logo from './logo.svg';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
 import './App.css';
+import Navbar from './components/Navbar';
+import MyProfile from './components/My-Profile';
+import Dragons from './components/Dragons';
+import MissionList from './components/Missions/MissionList';
+import Rockets from './components/rockets/rockets';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <Navbar />
```

Comment:
- [ ] Kindly ensure that the images are of equal dimensions and are aligned, as informed in the previous code review.

**Your design:**
![image](https://user-images.githubusercontent.com/12745474/188250471-35966389-00dc-43d7-8a86-a357d604dc27.png)


**The original design:**
![image](https://user-images.githubusercontent.com/12745474/188250528-64420992-88ee-4b0c-a7e1-d0a0f5f7942d.png)